src/styles/utils/FontUtils/index.ts

```diff
@@ -1,18 +1,15 @@
 import fontFamily from './fontFamily';
 import multiFontFamily from './fontFamily/multiFontFamily';
 import singleFontFamily from './fontFamily/singleFontFamily';
-import fontWeightBold from './fontWeight/bold';
+import fontWeight from './fontWeight';
 
 const FontUtils = {
     fontFamily: {
```

Comment:
Given our convo on Friday, I tried to simplify this but we need to be able to use `single` separately here, otherwise fonts won't be applied correctly on code blocks on Web. I will add comments on the code explaining each one 👍